en/contribute/README.md

```diff
@@ -1,23 +1,24 @@
-# Contributing
+# Community
```

Comment:
This is updated page to review. All the other changes are mostly prettier updates.